### PR TITLE
huobipro.py updated for fetch_orders() implement

### PR DESCRIPTION
**Pull request by tandazhi**
Referring same functions from okcoinusd and poloniex to implement the fetch_orders() and fetch_open_orders() for huobipro.

please review. Thanks.

### CODE REVIEW

**Comment by kroitor** [comment]
@tandazhi thank you! Merge in progress!

**Comment by kroitor** [comment]
References: #71

File: python/ccxt/huobipro.py [hunk 1]
```diff
@@ -279,7 +279,89 @@ def fetch_balance(self, params={}):
...
+        else:
+            raise ExchangeError(self.id + ' fetchOrders() requires type param or status param for spot market ' + symbol + '(0 or "open" for unfilled or partial filled orders, 1 or "closed" for filled orders)')
+        if status in [0,'open']:
+            status='submitted,partial-filled'
+        elif status in [1,'closed']:
+            status='filled,partial-canceled'
+        else:
+            raise ExchangeError(self.id + ' fetchOrders() wrong type param or status param for spot market ' + symbol + '(0 or "open" for unfilled or partial filled orders, 1 or "closed" for filled orders)')
+        response = self.privateGetOrderOrders(self.extend({
+            'symbol': market['id'],'states':status,
```
Comment (kroitor) [inline python/ccxt/huobipro.py]: I believe, `states` in this line is a typo, apart from that, everything seems ok, converting it to JS now.

Comment (tandazhi) [reply python/ccxt/huobipro.py]: please see below huopri API reference, 'states' is the parameter for the orders status. 
image

Comment (kroitor) [reply python/ccxt/huobipro.py]: This indeed looks like a typo in their docs, however, if you did test it and it worked ok, then this must be a typo in their engine API... Did it actually work with `statEs` for you?

Comment (tandazhi) [reply python/ccxt/huobipro.py]: yes, I have tested it. using 'states' is okay to fetch orders

Comment (kroitor) [reply python/ccxt/huobipro.py]: Got it, thanks for clarification! Merging this shortly.

**Comment by orkenstein** [comment]
@kroitor getting this after merge:
```javascript
[100%] Testing livecoin FAIL (testExchange @ run-tests.js:154)

FAILED livecoin (PHP): (explain @ run-tests.js:169)


	Parse error: syntax error, unexpected '=' in /Users/milo/Development/ccxt/php/huobipro.php on line 317 (explain @ run-tests.js:172)

FAIL ["livecoin"] (run-tests.js:250)
All done, 1 failed (run-tests.js:255)
```

**Comment by kroitor** [comment]
@orkenstein yeah, missed it somehow, uploaded a fix for it, thanks for pointing that out!

**Comment by orkenstein** [comment]
@kroitor 
```javascript
[100%] Testing livecoin FAIL (testExchange @ run-tests.js:154)

FAILED livecoin (PHP): (explain @ run-tests.js:169)


	Parse error: syntax error, unexpected '}' in /Users/milo/Development/ccxt/php/huobipro.php on line 321 (explain @ run-tests.js:172)

FAIL ["livecoin"] (run-tests.js:250)
All done, 1 failed
```

**Comment by kroitor** [comment]
Ok, tested it, should work now. Sorry for the inconvenience.

**Comment by orkenstein** [comment]
@kroitor no worries. It happens.

**Pull request merged**